mintapi/api.py

```diff
@@ -520,6 +516,29 @@ def _flatten_utilization(self, data):
                 )
         return utilization
 
+    def __find_endpoint(self, name):
```

Comment:
It seems like the endpoints is an array and each one has a "key", and then we have to find it inside. Is there a reason to not make ENDPOINTS a dict with a key so it actually is a key and you can just grab the rest of the values from it?

Reply:
@mrooney very good point! I can do that.

Reply:
@mrooney done!